### DISCUSSION TITLE
lineage os sync error

### DISCUSSION THREAD

**Discussion by youssefnone** [ROM-builders/temporary#17572] [General]
hi dev, thank you for giving developer this chance to maintain their device using this tools. so today i face an error when sync lineageOS version: lineage-17.1, the log given:
 error.GitError: Cannot checkout LineageOS/android_hardware_qcom_audio: LineageOS/android_hardware_qcom_audio rev-list ('^e20a6987ebc734a1e554836874da3b13383a2e4d', 'HEAD', '--'): fatal: bad object e20a6987ebc734a1e554836874da3b13383a2e4d
error: Cannot checkout LineageOS/android_hardware_qcom_audio
error: Unable to fully sync the tree.
error: Checking out local projects failed.
Failing repos:
hardware/qcom/audio
Try re-running with "-j1 --fail-fast" to exit at the first error.

**Comment by Apon77** [comment]
link?

**Reply by youssefnone** [reply]
did you fix the problem ?

**Reply by youssefnone** [reply]
did you fix the problem ?

**Comment by youssefnone** [comment]
> link?

https://cirrus-ci.com/build/5501900183830528
